cg/apps/mip/parse_sampleinfo.py

```diff
@@ -77,19 +88,19 @@ def parse_sampleinfo(data: dict) -> dict:
     Returns:
         dict: parsed data
     """
+    mip_sampleinfo = parse_obj_as(MipBaseSampleinfo, data)
     outdata = {
-        "date": data.get("analysis_date"),
-        "genome_build": get_genome_build(sample_info=data),
-        "case": data.get("case_id") or data.get("family_id"),
-        "is_finished": data.get("analysisrunstatus") == "finished",
+        "date": mip_sampleinfo.analysis_date,
```

Comment:
Suggest to return the model here instead of a dictionary

Reply:
done